src/app.controller.ts

```diff
@@ -1,12 +1,19 @@
 import { Controller, Get } from "@nestjs/common";
 import { AppService } from "./app.service";
+import { User } from "./entities/user.entity";
+import { UserService } from "./services/user.service";
 
 @Controller()
```

Comment:
please add UserController
that will called when you enter /user

@Controller("user")